omnigibson/simulator.py

```diff
@@ -1370,6 +1373,10 @@ def _deserialize(self, state):
             return self._scene.deserialize(state=state), self._scene.state_size
 
     if not og.sim:
+        
+        from omnigibson.systems.system_base import import_og_systems
+        # Import all OG systems from dataset
+        import_og_systems()
```

Comment:
This is kind of annoying in that it opaquely makes system load dependent on simulator launch but we dont say that anywhere. Let's reconsider this approach - we can discuss tomorrow.